docs/bgp-policy.md

```diff
@@ -93,6 +107,11 @@ provider to avoid issues caused by private ASN usage.
 The `listenPort` field specifies the port on which the BGP process listens. The default value is 179. The valid port
 range is `1-65535`.
 
+### Confederation
+
+The `Confederation` field specifies that the BGP process operates within a confederation identified by ASN `65000`,
```

Comment:
other fields are not capitalized
s/`Confederation`/`confederation`

Reply:
Done